lab_1_classify_by_unigrams/main.py

```diff
@@ -2,32 +2,33 @@
 Lab 1
 Language detection
 """
-
-
 def tokenize(text: str) -> list[str] | None:
-    """
-    Splits a text into tokens, converts the tokens into lowercase,
-    removes punctuation, digits and other symbols
-    :param text: a text
-    :return: a list of lower-cased tokens without punctuation
-    """
+    text = text.lower()
+    cleaned_text = ""
+    for symbol in text:
+        for letter in symbol:
+            if letter.isalpha() and symbol != " ":
+                cleaned_text += letter
+    tokens = list(cleaned_text)
+    if not isinstance(text, str):
```

Comment:
Looks like the text.lower() will assert error in case if text is not string

Reply:
Fixed